FFMQWebAsm/Pages/Index.razor

```diff
@@ -198,6 +198,18 @@
         }
     </Select>
 </Field>
+<Field>
+    <FieldLabel TextColor="TextColor.Secondary" class="fw-bold">Enemizer Attacks</FieldLabel>
+    <Tooltip Text="Shuffles enemy attacks. Safe = shuffle but don't add self destruct and Dark Knight attacks, chaos = shuffle and include self destruct and Dark Knight attacks, self-destruct = every enemy self destructs" Inline Multiline Placement="TooltipPlacement.Right">
```

Comment:
Use colon instead of equal sign for options, <br/><br/> for formating between options, options start with an uppercase letter and description ends with a period.